java-8/run-java.sh

```diff
@@ -16,8 +16,22 @@ set -x
 
 if test -f ./bin/app;
 then
-	./bin/app $@
+    ./bin/app $@
+elif test -f app.jar
+then
+    exec java \
+    ${DEFAULT_JVM_OPTS} \
+    ${JAVA_OPTS} \
+    ${PROXY_OPTS} \
```

Comment:
Innføring av ny proxy-variabel? Hvorfor? Proxy-settings kan settes via `JAVA_OPTS`

Reply:
Var ikke meningen at dette skulle være med i PRen, hang igjen fra tidligere kode.

Reply:
Ok, men hvorfor er ikke HTTP_PROXY godt nok?

Reply:
Ellers burde JAVA_OPTS bli utvidet med verdien av JAVA_PROXY_OPTIONS slik vi gjør med truststore noen linjer opp. Da funker det for alle tre kjøre-implementasjonene uten mer endring.

Reply:
`HTTP_PROXY` er ikke bra nok fordi den ikke oppdages av Java. Se kommentaren min her: https://github.com/nais/naisd/blob/90c8cbaa50b72d7495bdd7bfd0b02cb263b57a8a/api/resourcecreator.go#L423